frontend/Streamlit_GUI.py

```diff
@@ -0,0 +1,68 @@
+import streamlit
+from PIL import Image
+import os
+import requests
+
+
+streamlit.header("Steel surface defect analyser")
+number = streamlit.slider("Pick one of the 100 sample images")
+
+
+curr_path = os.getcwd()
+data_subdir_path = "/Dataset-trafo"
+data_path = curr_path + data_subdir_path
```

Comment:
Let's maybe refactor this:
```suggestion
data_path = f"{os.getcwd()}/Dataset-trafo"
```

Reply:
Changed to the suggested method.